### ISSUE TITLE
2nd or further comma separated selectors effect entire dom not just context node's children.

### ISSUE THREAD

**Issue by silverham** [Rct567/DomQuery#28]

**Comment by silverham** [comment]
Given:
```
$dom = new DomQuery('
<body>
  <div class="test"><p class="bob">111</p></div>
  <div class="test"><p class="bob">222</p></div>
</body>
');
$dom->find('.test')->first()->find('.someotherclass,.bob')->addClass('myextraclass');
// expected
<body>
  <div class="test"><p class="bob myextraclass">111</p></div>
  <div class="test"><p class="bob">222</p></div>
</body>

// produced.
<body>
  <div class="test"><p class="bob myextraclass">111</p></div>
  <div class="test"><p class="bob myextraclass">222</p></div>
</body>
```

**Comment by silverham** [comment]
Fixed in PR #29
